src/test/run-pass/abort-on-c-abi.rs

```diff
@@ -12,6 +12,7 @@
 // we never unwind through them.
 
 // ignore-emscripten no processes
+// ignore FIXME rust-lang/rust#48251 -- temporarily disabled
```

Comment:
This syntax appears to not be supported; we have to use `// ignore-test` instead I *think*...